src/content/api/batch-types/farmlands-external-asset.mdx

```diff
@@ -85,6 +85,10 @@ Exported Farmlands Credit Card information used for importing and updating of a
     Unique 9 digit field to display in barcode format to pay by [Patron Code](/api/patron-codes).
   </Property>
 
+  <Property name="number" type="PAN">
+    Unique 16 digit identifier for the card that can be used in conjunction with `expiry` for eCommerce payments.
```

Comment:
Suggestion: Simplify the description to say "16 digit primary account number of the card".
The PAN alone isn't unique. This is also decoupled from e-commerce so don't need to mention that implementation.